### ISSUE TITLE
(URGENT) PDF: Flipped Foreword and Introduction with a formally-specified Foreword clause

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-iso#918]
This is a PDF only issue.

In Adoption documents we need to have multiple Forewords, and they have to be encoded as "National forewords" using the following syntax:

```adoc
[.preface]
== National foreword

This British Standard is the UK implementation of ISO 44001:2017. It supersedes BS 11000-1:2010 which is withdrawn.
```

Via the `embed::` syntax, the NSB adoption needs to include the original ISO document. However, compilation fails if the ISO document uses the `.Foreword` syntax:

```adoc
.Foreword

ISO (the International Organization for Standardization) is a worldwide federation of national standards bodies ...
```

=> FAILS.

This means we have to use the formal Foreword clause syntax, i.e.
```adoc
[.preface]
== Foreword

ISO (the International Organization for Standardization) is a worldwide federation of national standards bodies ...
```

However, this results in a reversed Foreword / Introduction order in the document (only PDF), regardless of inclusion order (interesting because the Foreword is included before the Introduction, and alphabetically F goes before I):

PDF:
<img width="594" alt="Screenshot 2023-02-13 at 4 48 01 PM" src="https://user-images.githubusercontent.com/11865/218411941-12528be3-2ae0-4422-ae66-3855dfad5ef7.png">

HTML (no problem):
![Uploading Screenshot 2023-02-13 at 4.59.03 PM.png…]()

Word (no problem):
<img width="723" alt="Screenshot 2023-02-13 at 4 58 23 PM" src="https://user-images.githubusercontent.com/11865/218414305-d9bf6deb-26ac-443d-ba78-6e982259f653.png">

The document in question is BS ISO 44001 (added in https://github.com/metanorma/mn-samples-bsi/pull/111)
* ISO: https://github.com/metanorma/mn-samples-bsi/blob/main/sources/bs-iso-44001/iso-44001.adoc
* BS adoption: https://github.com/metanorma/mn-samples-bsi/blob/main/sources/bs-iso-44001/bs-iso-44001.adoc

This means the PDF processing code is doing some processing that is different from the desired presentation order. We should probably remove that.

**Comment by Intelligent2013** [comment]
There is a confusion. Very interesting issue.
1. mnconvert doesn't use `.Foreword` syntax, it puts always `== Foreword`. The documents in the folder `sources` encoded manually, The auto-generated by mnconvert documents - in the folder `sources.mnconvert`. For instance, similar structure in https://github.com/metanorma/mn-samples-bsi/tree/main/sources.mnconvert/adoptions/bs-iso-44002
2. I've generated presentation XML for https://github.com/metanorma/mn-samples-bsi/blob/main/sources/bs-iso-44001/iso-44001.adoc by the command `bundle exec metanorma -t iso -x presentation iso-44001.adoc `.
Presentation XML elements hierarchy is wrong (elements order and `displayorder` both):
```xml
<preface>
	<introduction id="_introduction" obligation="informative" displayorder="1">
		<title>Introduction</title>
	   ...
	</introduction>
	<foreword id="_foreword" obligation="informative" displayorder="2">
		<title>Foreword</title>
		...
	</foreword>
</preface>
```

3. mnconvert always put only
```
== Foreword
```
without `[.preface]`:

4. I've removed `[.preface]` and regenerate presentation xml again. The order is correct now:
```xml
<preface>
	<foreword id="_foreword" obligation="informative" displayorder="1">
		<title>Foreword</title>
		...
	</foreword>
	<introduction id="_introduction" obligation="informative" displayorder="2">
		<title>Introduction</title>
		...
	</introduction>
</preface>
```

PDF renders ok also:
![image](https://user-images.githubusercontent.com/5559926/218821945-d175aab0-1c63-4312-96aa-2fb79edca73d.png)
